### ISSUE TITLE
AX: Provide description to "Alt text" field on assets

### ISSUE THREAD

**Issue by AntonLecock** [statikbe/craft#396]
Can we add a description to the alt-text field on the assets?
Something along these lines – but it can definitely be shorter (or hidden behind a tooltip if possible):

"Alternative text is a textual substitute for images. Screen readers announce the alternative text, but it is also displayed if the image fails to load. Search engines also use alternative text and factor it into their assessment of the page purpose. Use the alternative text to describe what the image means within its context, not what it depicts. Leave empty if the image serves a purely ornamental or decorative purpose."

**Comment by SarahOorts** [comment]
Fixed met https://github.com/statikbe/craft/pull/413
